docs/preview/features/secret-store/named-secret-providers.md

```diff
@@ -7,9 +7,9 @@ layout: default
 
 The default workings of the secret store, is that a set of secret providers are registered and the consumer gets access to all of secrets provided by using `ISecretProvider`.
 
-In some cases, you may want to retrieve a specific secret provider from the store because that secret provider has some functionality that the other providers don't have.
+In some cases, you may want to retrieve a specific secret provider or a subset of secret providers from the store because that secret provider has some functionality that the other providers don't have.
 
-In those cases, you can register your secret provider with a unique name so that, in a later stage, you can retrieve back your named provider.
+In those cases, you can register your secret provider(s) with a name so that, in a later stage, you can retrieve back your named provider(s).
```

Comment:
```suggestion
In those cases, you can register your secret provider(s) with a name so that, in a later stage, you can retrieve your named provider(s).
```